### PR TITLE
Add Kitchen Etiquette

### PR DESCRIPTION
**Pull request by lukemorton**
We already had sink regime via RFCs but it’s time to add this to our office guides in the Handbook so new joiners are clued in.

### CODE REVIEW

File: guides/office/kitchen.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have two thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
```
Comment (cybojenix) [inline guides/office/kitchen.md]: I believe there are 3 jugs now

File: guides/office/kitchen.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
```
[the remark this answers is not in the document]
Comment (lukemorton) [reply guides/office/kitchen.md]: Sorry Dan :( You can add another PR to add some etiquette around tea?

File: guides/office/kitchen.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have multiple thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
+
+ - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
+ - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
```
Comment (DelphineGattaz) [inline guides/office/kitchen.md]: can we have a rule that if you take the last cup of coffee in the pot you need to make a new pot? but not if it is later in the day, say after 15.00?

Comment (StevenLeighton21) [reply guides/office/kitchen.md]: _kill the joe, make some mo'_

File: guides/office/kitchen.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Kitchen Etiquette
+
+We share the kitchen space with each other as well as with our customers. We have put together a few guidelines for keeping the kitchen nice and tidy.
+
+## Coffee
+
+We love our coffee at Made Tech. We have fresh beans regularly delivered to the office and have multiple thermos drip coffee jugs that keep coffee lovely and fresh. Please ensure:
+
+ - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
```
Comment (StevenLeighton21) [inline guides/office/kitchen.md]: Is the cupboard trick acceptable?

File: guides/office/kitchen.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+ - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
+
+## Sink regime
+
+We all appreciate a clear and useable sink. Please ensure:
+
+ - When you're finished with glasses, cutlery and crockery that you wash them up
```
Comment (adrianclay) [inline guides/office/kitchen.md]: Don't believe this happens at the moment, would mean a lot of washing up happening at the end of the day.  I'd be pro this nonetheless.

File: guides/office/kitchen.md (hunk 3 above)
Comment (lukemorton) [reply guides/office/kitchen.md]: While I'm the biggest fan of the rule (and video on youtube) I don't think we should codify it.

File: guides/office/kitchen.md (hunk 4 above)
Comment (lukemorton) [reply guides/office/kitchen.md]: I think batch grind at beginning of day more acceptable.

**Pull request merged**